tasks/main.yml

```
@@ -31,6 +31,16 @@
     backup: true
     mode: "{{ __kdump_conf.stat.mode | d('0644') }}"
   notify: Restart kdump
+  when: ansible_facts['distribution'] in ['RedHat', 'CentOS', 'Fedora']
```

```suggestion
  when: ansible_facts['os_family'] == 'RedHat'
```
Rather than trying to enumerate all of the possible "RHEL-like" OSes.